contracts/libs/RatioUtils.sol

```diff
@@ -19,7 +19,7 @@
 pragma solidity ^0.8.0;
 
 import { SafeMath } from "@openzeppelin/contracts/utils/math/SafeMath.sol";
-import "hardhat/console.sol";
+import "hardhat/console.sol"; // TODO:
```

Comment:
this one is left

Reply:
i left it not to forget to remove it in the future, but, i guess, shouldn't be an issue if we remove it now